src/Android/Xamarin.Android/Samples/Map/GetElevationAtAPoint/GetElevationAtAPoint.cs

```diff
@@ -0,0 +1,156 @@
+        }
+
+        private void Initialize()
+        {
+            // Create the camera for the scene.
+            Camera camera = new Camera(_observerPoint, 20000.0, 10.0, 70.0, 0.0);
+
+            // Create a scene.
+            Scene myScene = new Scene(Basemap.CreateImageryWithLabels())
+            {
+                // Set the initial viewpoint.
+                InitialViewpoint = new Viewpoint(_observerPoint, 1000000, camera)
+            };
+
+            // Create the marker for showing where the user taps.
+            _elevationMarker = SimpleMarkerSceneSymbol.CreateCylinder(Color.Red, 10, 750);
+
+            // Create the text for displaying the elevation value.
+            _elevationTextSymbol = new TextSymbol("", Color.Red, 20, Esri.ArcGISRuntime.Symbology.HorizontalAlignment.Center, Esri.ArcGISRuntime.Symbology.VerticalAlignment.Middle);
+            _elevationTextGraphic.Symbol = _elevationTextSymbol;
+
+            // Create the base surface.
+            _baseSurface = new Surface();
+            _baseSurface.ElevationSources.Add(new ArcGISTiledElevationSource(_elevationUri));
+
+            // Add the base surface to the scene.
+            myScene.BaseSurface = _baseSurface;
+
+            // Graphics overlay for displaying points.
+            _overlay = new GraphicsOverlay
+            {
+                SceneProperties = new LayerSceneProperties(SurfacePlacement.Absolute)
+            };
+            _mySceneView.GraphicsOverlays.Add(_overlay);
+
+            // Add the scene to the view.
+            _mySceneView.Scene = myScene;
+        }
+
+        private async void SceneViewTapped(object sender, Esri.ArcGISRuntime.UI.Controls.GeoViewInputEventArgs e)
+        {
+            try
+            {
+                // Remove this method from the event handler to prevent concurrent calls.
+                _mySceneView.GeoViewTapped -= SceneViewTapped;
+
+                // Check that the point is on the surface.
+                if (e.Location != null)
+                {
+                    // Clear any existing graphics from the graphics overlay.
+                    _overlay.Graphics.Clear();
+
+                    // Get the elevation value.
+                    double elevation = await _baseSurface.GetElevationAsync(e.Location);
+
+                    // Set the text displaying the elevation.
+                    _elevationTextSymbol.Text = $"{Math.Round(elevation)} m";
+                    _elevationTextGraphic.Geometry = new MapPoint(e.Location.X, e.Location.Y, e.Location.Z + 850);
+
+                    // Add the text to the graphics overlay.
+                    _overlay.Graphics.Add(_elevationTextGraphic);
+
+                    // Add the marker indicating where the user tapped.
+                    _overlay.Graphics.Add(new Graphic(e.Location, _elevationMarker));
+                }
+
+                // Re-add to the event handler.
+                _mySceneView.GeoViewTapped += SceneViewTapped;
```

Comment:
I'd put this in a `finally` block. That will ensure the event still works after an exception.